test/common_utils.py

```diff
@@ -1111,15 +1111,42 @@ def random_fullrank_matrix_distinct_singular_value(matrix_size, *batch_dims,
     dtype = kwargs.get('dtype', torch.double)
     device = kwargs.get('device', 'cpu')
     silent = kwargs.get("silent", False)
+    singular = kwargs.get("singular", False)
```

Comment:
I think that introducing this is contradictory to the name of the function.

Reply:
gotcha, I will fix the naming.

Reply:
Fixed